specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-12-10-preview/examples/HostPool_Update.json

```diff
@@ -17,7 +17,7 @@
         "loadBalancerType": "BreadthFirst",
         "personalDesktopAssignmentType": "Automatic",
         "registrationInfo": {
-          "registrationTokenOperation": "Update"

```

Comment:
you also need "registrationTokenOperation": "Update" to update.

Reply:
Hi @datr93 , thanks for review.

I got 500 error when I try to PATCH with  "registrationTokenOperation": "Update",  so I had thought the example isn't correct. Would you help on it?

And I find that: if I don't update registrationTokenOperation in a PATCH, it will be OK.

~~~
Request method: 'PATCH'
Request headers:
    'Content-Type': 'application/json'
    'Accept': 'application/json'
    'Content-Length': '234'
    'x-ms-client-request-id': '5710bdc2-9b0c-11ea-8d2a-70bc107fb75f'
    'CommandName': 'desktopvirtualization hostpool update'
    'ParameterSetName': '--name --description --friendly-name --load-balancer-type --max-session-limit --tags --resource-group --registration-info --debug'
    'User-Agent': 'AZURECLI/2.5.1 (PIP) azsdk-python-mgmt-desktopvirtualization/0.2.0 Python/3.8.1 (Windows-10-10.0.18362-SP0)'
    'Authorization': '*****'
Request body:
{"tags": {"tag1": "value1", "tag2": "value2"}, "properties": {"friendlyName": "friendly", "description": "des1", "maxSessionLimit": 99, "loadBalancerType": "BreadthFirst", "registrationInfo": {"registrationTokenOperation": "Update"}}}



Response status: 500
~~~

Reply:
You need both to update the token. And in this case update means create a new token.

"registrationInfo": {"registrationTokenOperation": "Update", "expirationTime": "2020-10-02T14:01:54.9571247Z"}

Reply:
Hi @datr93, thanks for help. I've just commit the change about it in swagger and example.